### PR TITLE
David loran remove video

### PR DESCRIPTION
**Pull request by Aholicknight**
this video is bad and cringe and should be de*yeet*ed

Issue: #329

### CODE REVIEW

File: www/artists/dloran/artist.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "location": "Regina",
   "changelog": [
     "Created by Mark Romanow: 1996."
```
Comment (synixebrett) [inline www/artists/dloran/artist.json]: The build is failing because you need a comma at the end of this line.

Comment (trollmad3) [reply www/artists/dloran/artist.json]: What line would I put the comma on? I always have this problem when I edit the `artist.json` file.

Comment (synixebrett) [reply www/artists/dloran/artist.json]: The line I commented on, line 5.

File: www/artists/dloran/work.html [hunk 2]
```diff
@@ -57,14 +57,9 @@
                     </div>
                 </div>
                 <div class="c1">
```
Comment (synixebrett) [inline www/artists/dloran/work.html]: This div should probably be removed entirely

File: www/artists/dloran/index.html [hunk 3]
```diff
@@ -30,10 +30,10 @@
         </div>
         <div class="footer">
             <p>
-                Images are copyright of the Artist
+            Images are copyright of the Artist
```
Comment (synixebrett) [inline www/artists/dloran/index.html]: This should be indented as it was

File: www/artists/dloran/index.html [hunk 4]
```diff
@@ -30,10 +30,10 @@
...
             </p>
             <p>
-                Created by Mark Romanow
```
Comment (synixebrett) [inline www/artists/dloran/index.html]: This should be indented as it was

File: www/artists/dloran/index.html (hunk 3 above)
Comment (synixebrett) [reply www/artists/dloran/index.html]: Yes, it needs to be pushed back to the right

**Pull request closed without merging**